### ISSUE TITLE
For older models - Model LM40SAMFHD700AG25WV

### ISSUE THREAD

**Issue by avitascmor** [olm3ca/mirror#8]
Here is the [technical manual](https://m.panelook.cn/LM40SAMFHD700AG25WV-40-TFT-Liquid-Crystal-Display-module-with-LED-Backlight-unit-detail_155306.html)

I believe this is one of the earlier models that requires a specific cable and controller: https://ebay.us/HkicYV

This controller and cable set work and provide VGA, DVI and HDMI inputs for a Pi, Laptop, etc.  I only replaced the LVDS cable and let the rest of the screen remain powered by design.  I have yet to add the sound to the new board but the connections are there.

Here it is powered up with my laptop:

https://imgur.com/gallery/e8AqDlA

With MagicMirror2 running on a Pi4

https://i.imgur.com/MrhZ6gI.jpeg

Hope this helps everyone out!

**Comment by olm3ca** [comment]
This is excellent news, thank you @avitascmor. I'll add this to the instructions today! Really great you found that controller board, thanks for sharing.

**Comment by grhill424** [comment]
@avitascmor 
I ordered the driver. I was wondering if you could send some pictures of the inside, how you hooked it up and drew power etc. Thanks so much in advance. I am excited to get this mirror working again.

**Comment by minihadroncollider** [comment]
@avitascmor 
Great job setting this up! I would also appreciate some pictures or any guidance you have on setting up the LM40SAMFHD700AG25WV panel. I have a "Model One" with this panel and am waiting on delivery of the controller.

**Comment by olm3ca** [comment]
I would assume you just need to connect LVDS and provide 12V power to the new board, using an external plug. It also looks like you can connect audio in a similar way to how I did it with the Vizio board. @grhill424 did you get yours working yet?

**Comment by avitascmor** [comment]
Yes on the audio that should not be an issue. The power I wanted to see if I could draw from internal anywhere vs. an external adapter. The inverter/backlight just works fine as powered.
________________________________

**Comment by grhill424** [comment]
@olm3ca i am still waiting on delivery of the controller. I will update once it arrives and I (hopefully) get it up and working.

**Comment by jeremy-fields** [comment]
I'm having trouble getting my LM40SAMFHD700AG25WV Mirror into initial setup mode. I can't find a button to place it into factory reset. Externally visible, I have a power input, ethernet, USB, microphones, and a power switch. Pics of my mirror. https://imgur.com/a/KXOlqAa 
The only internal buttons I see are power and volume up/down, all on the scalar board. I did hold the power button down for ~10 and ~20 seconds. Both attempts resulted in the mirror rebooting after releasing the button. It then boots up and connects to my home wifi like before - it doesn't appear to have reset. I also tried holding each volume button ~10 seconds independently with no effect. Didn't try any combo button presses.

**Comment by avitascmor** [comment]
I thought as it is android based it will be either hold down volume up or down and keep it pressed while you press Power. Keep the volume pressed until the boot process starts.
________________________________

**Comment by jeremy-fields** [comment]
I've tried a few different combinations. A lot of the factory reset websites have you shutting it off first. I can't really turn it off. I can pull power, but shutting it down has been troublesome. I can reboot it and then try the Power + Volume Up combo which makes it switch to a "Update Complete!" display (see imgur link). It stays here until I reboot again. After that reboot it looks normal, and connects to my Wi-Fi as usual. I don't what software version was on there before but it's currently showing v.1.20.0. If I leave it in this post-boot state, the backlight eventually dims. I can't restore the backlight with the volume buttons, only another reboot. Holding the power button just reboots after about 15 seconds. Holding Power + Volume Down results in what appears to be a constant reboot state. The backlight comes on and off, I saw the Mirror logo pop up once very briefly, and then the backlight just continues to cycle. This continues until I release the buttons.
https://imgur.com/a/Vu8vdUI

**Comment by mweth** [comment]
I bought this controller and am getting it setup.  It seems like there should be a better way to add power to the controller and screen than using the old board.

**Comment by avitascmor** [comment]
I’ve had no time to tap into the voltages to see if the LVDS board could be powered from the existing power supply to negate an external one. That would be my preferred option. Should be able to tap into something to power a Pi also so it’s all self contained in the mirror.
________________________________

**Comment by mweth** [comment]
Here is the power supply for anyone smarter than me:

https://www.posturk.com%2FB2bDosyalar%2FNesne%2FMP118TX.pdf

**Comment by jeremy-fields** [comment]
I was finally able to get mine into what looks like a reset mode where the backlight stays on. I disconnected the Wi-Fi RF connector from the board. 
The mirror was previously on my network and would rejoin when booted. I've tried resetting it (see above). I also tried through the app, but we cancelled the service so it wouldn't even let me log in with my account since my wife was the primary owner. She has since deleted the app. I didn't try the app any further. I'm not sure if the above reboot plus Power & Volume Up combo contributed? 
With the RF disconnected, it boots up, tries to find the wifi, can't and gives me the setup prompt and just stays there in it's backlit glory that I've been searching for. I had it up for 5-10 mins. I'll try longer and reply if I see different results. I tried to connect to the Mirror's setup Wi-Fi but I don't see it; disconnecting the antenna seems to have either turned it off or dropped the signal so it couldn't possibly connect again.

<img src="https://github.com/olm3ca/mirror/assets/87494419/550d13f8-e2cd-4fc9-98b3-8fe380cecdaf" width="300" height="400">

<img src="https://github.com/olm3ca/mirror/assets/87494419/ef46c038-b8d9-456a-97e2-9e6e5a7b3395" width="300" height="400">

<img src="https://github.com/olm3ca/mirror/assets/87494419/d0581d3b-9b4a-42ae-b53a-0794ade39d17" width="300" height="400">

<img src="https://github.com/olm3ca/mirror/assets/87494419/0cfab7eb-640c-4594-94ae-203a85ce64c9)" width="300" height="400">

**Comment by olm3ca** [comment]
Glad you figured that part out, @jeremy-fields - keep us posted on the rest of the setup. For the repo, what steps would you suggest for other users with the same panel for how to keep the backlight on?

**Comment by jeremy-fields** [comment]
I wasn’t sure how to get the Mirror into an initial setup mode without the button, so I tried a lot of Android rest options. I’m not sure if they contributed, but the key for me seems to be getting the Mirror off the network. 

1. Reset the Mirror to initial state through the app or by pressing the blue button if installed.
2. If you cannot complete the above step and the Mirror is connected to Wi-Fi, remove it from your network by either kicking it off from the router, changing the Wi-Fi password, or removing the RF cable connecting the Wi-Fi antenna to the mainboard (RF cable removal may be irreversible).

I ran it for just over an hour yesterday and the backlight stayed on the whole time. I have a Pi and I'm currently waiting on a new Mainboard to arrive.

**Comment by olm3ca** [comment]
@jeremy-fields thanks! Added some additional details to the instructions. Let us know how it goes when the mainboard arrives.

**Comment by jeremy-fields** [comment]
Build complete

I pulled out the original microphone board, usb access, video board, and speaker amplifier. I installed the [new board](https://www.ebay.com/itm/166580220209?siteid=0&customid=lululemon&toolid=20012) from eBay, a Raspberry Pi 4, and some connectors. I left the camera board and lens installed to prevent an obvious hole visible from the front (could also tape it off but this was easy). I also left the Power Supply and Backlight Inverter in place.

Mounting the boards was a little bit of a challenge. I used two posts from the speaker amp and two 0.5" [sticky mounts](https://www.amazon.com/dp/B07F23656R?psc=1&ref=ppx_yo2ov_dt_b_product_details ) on the Pi. I had to mount the Pi to a [mounting board](https://www.amazon.com/dp/B07BX2BVWB?psc=1&ref=ppx_yo2ov_dt_b_product_details) first.  

I used 4 [shorter sticky mounts](https://www.amazon.com/dp/B0C7QQ97GG?ref=ppx_yo2ov_dt_b_product_details&th=1) (0.25") for the new monitor board. The LVDS cord length and PSU cord lengths limit placement.

I added a USB-A port to the top of the mirror for the Pi. There was a USB micro there originally, but I wanted something easy to plug into so I swapped it out with [this](https://www.amazon.com/dp/B00G6SOP3I?ref=ppx_yo2ov_dt_b_product_details&th=1). From where my Pi was mounted, I had to run a USB extension cord to it.

I drilled a 1" hole in the power/network/connector box in the right spot to add a USB-C connection for powering the Pi. I used [this bit](https://www.homedepot.com/p/Milwaukee-1-in-Hole-Dozer-Bi-Metal-Hole-Saw-with-3-8-in-Arbor-Pilot-Bit-49-56-9662/202327732) from Home Depot. I added [this part](https://www.amazon.com/dp/B0CDPTQDJ8?ref=ppx_yo2ov_dt_b_product_details&th=1) for a USB-C external port. They make [USB-C and USB-A combos](https://www.amazon.com/dp/B09TPC72Y9?psc=1&ref=ppx_yo2ov_dt_b_product_details), which I tried, but my drill hole was too small so I just used the USB-C bulkhead connector. It's pretty full down there, so maybe not a bad choice.

I was a little worried about the boards being so close to the cover so I added some Kapton type tape. Might be overkill. I chose [this](https://www.amazon.com/dp/B072Z92QZ2?psc=1&ref=ppx_yo2ov_dt_b_product_details) which came with way more than I needed.

The new board can power the backlight if you connect the wiring per the included instructions. I did that and was able to completely remove the old board. I wrapped all the new wires near the Power Supply with 80 degree C electrical tape. The original wires had a coating over them with 80C listed on it. I originally tried to power the board through the power plug, but wiring it direct seems to be the better choice. I did had to move some pins around to get the plugs right. If you do this, pay attention to the +/- because the colors are not always correct - I found red wires connected to negative connections. 

Since I only have one USB-A external connection, I bought [this hub](https://www.amazon.com/dp/B0BXWV79TK?ref=ppx_yo2ov_dt_b_product_details&th=1) to connect my keyboard and mouse to. It's nicer to be in front of the mirror when configuring it versus using VNC since the monitor is turned sideways (the mouse is hard on VNC). 

The backlight is probably too bright for having this in the bedroom, so I bought a managed power outlet. I like the Kasa plugs so I bought [this](https://www.amazon.com/dp/B07B8W2KHZ?ref=ppx_yo2ov_dt_b_product_details&th=1). You can schedule it to turn on and off at certain times. 

Wi-Fi might be hard with the Pi inside, so I opted for a Ethernet connection. 
I haven't sorted out audio yet. The video board has audio connectors that are smaller than anything I pulled off, so pinning has been difficult. I did rig something up at one point, and it sounded good, but the wires don't really stay connected so I removed it for now. I might come back to it later.
The new video board came with a control board, but I removed it. You'd need to run it outside somehow and I don't think you really need it.

Now to figuring out how I want the software configured. 

Pictures: https://imgur.com/a/PxCpFC0

![New overview](https://github.com/olm3ca/mirror/assets/87494419/cc2112d8-aa69-4797-8176-29bfa287006e)

![Running 2](https://github.com/olm3ca/mirror/assets/87494419/f389971a-311f-4499-a1b0-c1c3160f2608)

**Comment by olm3ca** [comment]
@jeremy-fields fantastic work, so glad to see it working! I'll add a how-to section for your board with the notes you provided. A lot of folks with this model of the Mirror will benefit from this.

**Comment by minihadroncollider** [comment]
@jeremy-fields
Many thanks for this fantastic write up. I am following your guide closely, making a few little mods along the way. I have a couple of questions, if you don't mind...
Firstly, I'm curious on the repinning you had to do for hooking up the power to the board. I'm assuming this was to connect the external power switch as everything else on my board seems to match with the instructions you included. Did you have to do any other repinning?
Secondly - did you get anywhere with the audio? I'd really appreciate it if you took any pictures of the rig up you made so that I might start there and see if I can troubleshoot it.

**Comment by jeremy-fields** [comment]
There is no reason to re-pin unless you want to use that power switch. The standard loopback cable keeps the board in a switched-on mode. I wanted it, so I re-pinned it. It's probably not worth the hassle if you plan to turn it on & off some other way. I don't think any re-pinning is necessary unless you want that switch.

I stopped looking at the audio for now. I'm running [magicmirror](https://magicmirror.builders/) and it's full screen. There may be an audio/video module but I haven't looked for one. What I tried was to splice the old speaker cords into a longer wire with pins (leftover cords I pulled off), then force/connect that to the speaker connector on the new board. It worked, but the cables were heavy and pulled off easily since I just had them slid on. With a proper connector, it seems like an easy add. I don't know what connector to buy. There's also a headphone jack (says EARPHONE IN on the back) if you want to try to splice a cord for that. 
Here's some pics: https://imgur.com/a/ltg9ejt

**Comment by i3randnew** [comment]
First off -- big thank you to everyone on this thread. It's been an extremely fun project to build off of, including all of the exploration upfront on the board to purchase and various setups.

@minihadroncollider I ended up with a repinned solution myself for both power and for audio that has worked nicely for the time being. I am still waiting on my smart plug to turn the Pi and display on/off, so right now I end up manually shutting down the pi and flipping off the display, but it works well and I didn't have to bore any additional holes in the Mirror casing which I was reticent to do.

For audio, the [pics](https://imgur.com/a/ltg9ejt) from @jeremy-fields are great and show exactly what the connector is, what the pins are mapped to, and generally what female connector you would need (mini micro jst). I eneded up going with repurposing two cables, splicing them to the speaker cables, and repinning the two cables (4 pin connections) to a connector from a [mini micro JST 2.0 connector with 2mm/0.078 inch pitch](https://www.amazon.com/dp/B01DUC1S7S). This ended up working nicely and get great sound without needing to solder or having the connections fall out.

For power, I went back and forth on this, but really wanted to power the pi from the main power supply in the Mirror. I went down a rabbit hole and there's still some low power warnings on my pi 5 about peripherals, but it is working really well so far and I haven't noticed any issues in performance on both the pi or the wifi adapter (more below). To get this to work, I bought [Dupont and JST connectors and a crimper](https://www.amazon.com/dp/B0C6PM9KGY) to create solid connections to the pi GPIO pins, 2 5v and 2 ground connectors from the main power supply board. The board is very clear on where the cables need to be pinned for each connection. 

For my board, I went with a [slightly different version](https://www.ebay.com/itm/235438526466) than what was originally discussed on the thread because I wanted USB-C / DP / HDMI for my connections vs VGA / DVI / HDMI. They're nearly identical other than the connections, so the audio pinning was similar. 

For internet, I hooked up the wired ethernet from the pi to the cable / port already on the Mirror, but I really didn't want to run a wire to the ethernet port. Since the Mirror casing is solid steel, the wifi connection was going in and out depending on the day. To solve for this, since I bought the [same USB extension as mentioned above](https://www.amazon.com/dp/B00G6SOP3I) and connected it to the pi and mounted it to the top of the mirror where the previous micro-usb port was mounted, I was able to buy a [nano usb wifi adapter](https://www.amazon.com/dp/B09KTDXPY3) and [install the drivers](https://community.tp-link.com/en/home/forum/topic/208022) really easily. Wifi is now extremely reliable, fast, and you can barely tell it's there.

For software, I am running magic mirror right now as well, but am working on how to stream workouts. There are a few solutions regarding mirroring, but nothing I have found that's a straight solution for Apple Airplay from individual apps. If anyone has a pointer or advice, I would love to hear how you solved this. 

Here's some pics / descriptions of my setup: https://imgur.com/a/eJUqtcT

**Comment by minihadroncollider** [comment]
@jeremy-fields Many thanks for the quick reply! I will keep the power switch, so will go about repinning that. I'm glad to hear I didn't miss anything else. My plan is to run Magic Mirror on a Pi 5 (probably overkill) in kiosk orientation. I appreciate the pictures of your successful audio connection, albeit not stable. I'm sorry if that meant opening up the Mirror again!

@i3randnew Also many thanks! I'll follow your lead for the audio. I may bust out the soldering gun and heat shrink tubes to get a nice clean install.
I have been worried about the wifi, so might just try what you've done as I don't want to pull more network cables in the wall cavity. Were the Ubuntu drivers that you posted suitable for the pi OS?
Regarding mirroring, I had assumed that installing [RPiPlay](https://github.com/FD-/RPiPlay) on the pi 5 would allow you to use Apple Airplay. I had read that the Magic Mirror software just defaults to the Airplay while it's in progress and then back to the Magic Mirror when stopped. I haven't set up my pi yet, though - so I'm only speaking from browsing on the matter. If you've already tried this, do let me know if you find a solution. I want my Mirror to do Magic Mirror, workouts and media (both from the pi and AirPlay/Chromecast).

Once again, many thanks to everyone here. I am only able to do a little bit on this every week after hours, so my progress is slow - but the instructions, advice and tips I've gleaned here have been invaluable. I even bought a second Mirror at a heavy discount to make a second one after this one is done.

**Comment by olm3ca** [comment]
I use uxplay for Airplay streaming and it’s excellent. I mostly use it for
audio but video also works. The pi 5 should be able to handle video quite
well.

**Comment by i3randnew** [comment]
Ah, thanks! I took a look at uxplay as well -- but found the below which isn't quite what I am looking for. I've gotten AirPlay mirroring to work just fine using the RPiPlay, but I want streaming as the quality tends to suffer with display mirroring. I'd like to have the same streaming experience that AirPlay2 gives. I'll keep searching.

> Originally supported only AirPlay Mirror protocol, now has added support for AirPlay Audio-only (Apple Lossless ALAC) streaming from current iOS/iPadOS clients. **There is no support for Airplay2 video-streaming protocol, and none is planned.**

**Comment by jeremy-fields** [comment]
Thanks for the audio connector @i3randnew. The build pics are cool to see too! I love the usb-wifi idea, simple and brilliant. I've been struggling with what to do if I moved mine from the location with wired network. I did [find a person](https://www.ebay.com/itm/114773307539?var=414932137978) who solders a WiFi RF cable connector onto a Raspberry Pi. I bought one, but I'd need to move my pi to the top of the mirror where the old board had it's wifi "window". 

For my USB power passthrough on the bottom, I removed the power sub-piece steel section and drilled it by itself. I didn't want to drill anything in the main structure with the monitor in it (no vibrations there). Wasn't too hard to remove and then drilling was easier, although I used a drill press which made it not scary - overkill I guess.

@minihadroncollider I had a lot of pictures, I just posted some old ones. Unrelated, I did open it up this weekend. My Pi4 was running a little hot at times. It got over 70C a couple times when I was messing with updating the magicmirror configs over and over. It functioned ok but worried me some so I added a fan and heat sink. It's never over 51C now without the fan even kicking on; the fan is programmed to come at 60C.  I bought [this fan](https://www.amazon.com/dp/B091L1XKL6?psc=1&ref=ppx_yo2ov_dt_b_product_details) from Amazon.

Fan and power panel pics https://imgur.com/a/6zGZNGR

**Comment by cobra85ec** [comment]
Do this board work with [LTI400HN01](https://www.panelook.com/LTI400HN01_Samsung_40_LCM_parameter_31646.html)

**Comment by rooibos-hommes** [comment]
> Do this board work with [LTI400HN01]

> Do this board work with [LTI400HN01](https://www.panelook.com/LTI400HN01_Samsung_40_LCM_parameter_31646.html)

I don't think so. I am trying different off-the-shelf boards. I tried [this one](https://www.alibaba.com/product-detail/M53-V2-0-board-kit_1600759994284.html) already and got bad results [shown here ](https://github.com/olm3ca/mirror/issues/9). Next, I plan to try [one of these boards](https://jx-lcd.en.alibaba.com/search/product?SearchText=JX-V53-UHV). however still not too sure which has a better chance at working.

**Comment by minihadroncollider** [comment]
@i3randnew and @jeremy-fields 
I've finally found some time to finish the project off while the family are traveling. Everything has gone to plan so far. I have the screen working with sound and the original switch turns the board on and off. Your instructions have been most helpful.

@i3randnew - do you have any more details or close up pictures on how you powered the Pi 5 from the power supply board? I have the DuPont crimper and JST connectors, but I can't work out quickly how to connect it all up - it's not something I've done before. What did you refer to for the board and the Pi to connect the wires correctly? Any advice would be appreciated as once I have that done, it's just tidying things up and then working on the MagicMirror set up.

**Comment by minihadroncollider** [comment]
@i3randnew - actually, I did a little research and it seems pretty self-explanatory in the end. I'll give it a shot tomorrow and see if I can get it working.

**Comment by toxict0ast** [comment]
Great work everyone! I'm waiting on all my parts to come in. Excited to upgrade my current smart mirror setup with this mirror. I got one for 100$ on FB marketplace. 

I'm having some issues getting account switched over to me to do the factory reset. Is it really necessary? 

@minihadroncollider do you have a photo of where you connected the 5v power for the pi? I understand the power inputs on the GPIO side, but not where to connect it on the main board.

**Comment by minihadroncollider** [comment]
I meant to do this a month ago, but I was hit by a hurricane and have then been out of town. In any case, here are soe pictures and descriptions of my set up: https://imgur.com/a/lZcJO51

@jeremy-fields and @i3randnew - you will see I have used your guidance. Thank you so much - I could not have done this without your posts.

@toxict0ast - hopefully this give you what you need to power the Raspberry Pi. I have included a good guide for the pins that was helpful for me. The factory reset is necessary for the TV backlight to stay on permanently - I would try your best to do this before you start.

The only problems I'm having right now are:
1. I get low power warnings, just like @i3randnew mentioned - but they don't seem to affect anything, so I just ignore them.
2. I haven't worked out a way to get the Pi to default to the external wifi dongle each time it's turned on.

Things I want to work on now it's done:
1. Setting up the MagicMirror software with some good and useful modules
2. Working out how to shut the Pi down quickly before I press the switch at the bottom of the Studio Mirror
3. Adding AirPlay functionality so that a iPhone can stream to the Mirror
4. Seeing if I can get a presence sensor working with the screen to turn it on and off
5. Connecting a Google TV dongle for watching shows, but with vertical orientation

**Comment by toxict0ast** [comment]
Amazing work @minihadroncollider ! Thank you very much, those photos help so much. 

Sadly it seems there isn't a way for me to get the account switched to me for a factory reset. I will still try this project, but it doesn't sound hopeful. I have all the parts delivered this week for some testing.

**Comment by olm3ca** [comment]
@minihadroncollider great update! You've documented this well. For Airplay, I've been a fan of uxplay. 

Can Google TV rotate orientation?

**Comment by toxict0ast** [comment]
I finished the project!
I just wanted to report back a few things.

1. It doesn't seem to matter that I didn't do a factory reset on the 2nd hand mirror. It was never connected to my network also. The screen has been on a solid 24 hours. 

2. I was able to power the Pi from just one 5v positive, and 1 ground pin. I used the other +5volt to power my fan, because i'm using a pi4, and it doesnt have dedicated pins for fans. 

Thanks for all the help on these projects. The pictures posted were SO MUCH help. Yall are awesome and I love this mirror.

**Comment by tung256** [comment]
@i3randnew could you please take a look at my post here https://github.com/olm3ca/mirror/issues/15
i took closeup shots because your photos and descriptions were not clear. thanks a lot for your time and help!

**Comment by tung256** [comment]
@toxict0ast 
this is the board i got https://www.ebay.com/itm/235438526466?var=535565344805

the NEW and OLD video cables dont match, can i just plug the new cable in?
![image](https://github.com/user-attachments/assets/475e765c-928e-4ce0-b93b-7874feb7b72a)

how are you powering up the new board?  the old board has 4 wires for power via G,G,24,24

for backlighting, can i plug NEW1 into the panel? do i have to change any pins?
![image](https://github.com/user-attachments/assets/174b09da-6c54-4984-838b-5cea845b81d3)

**Comment by tung256** [comment]
@jeremy-fields mine did came with manual but very blurry. your manual in the last 2 images in the link is basically identical to mine! this is good.  your [2nd link ](https://github.com/olm3ca/mirror/issues/10#issuecomment-2076426623) does have very good photos BUT some images i cannot zoom and enlarge. when i click, it shows up as "private." please change when you can as these photos are very crucial.

ok. so to confirm...i power the backlight using the new1.  new2 and new3 actually goes to the Mirror's Power Supply board, correct?
new board does indeed already have 4th piece power cable already connected to the board. in manual, it is labeled CN5.  all this time, i was trying to figure out how you powered up the new controller using the old power connections (ground, ground, 24, 24)

thanks so much for confirming everything.

**Comment by jeremy-fields** [comment]
I'm not sure about the pictures not showing. I logged out of github and they still work for me. The [URL](https://private-user-images.githubusercontent.com/87494419/325483159-6fc514fc-c7b1-4ac1-91b9-672f7e28c7f3.jpeg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQ5NDA2NzksIm5iZiI6MTcyNDk0MDM3OSwicGF0aCI6Ii84NzQ5NDQxOS8zMjU0ODMxNTktNmZjNTE0ZmMtYzdiMS00YWMxLTkxYjktNjcyZjdlMjhjN2YzLmpwZWc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwODI5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDgyOVQxNDA2MTlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT01YWE3YTU3ZjkxYjNmZDUxMWMxNTlmMzM1Njc4ZjY3ZWM4MjE4MGI2MDA3ZWUwZmVjMGRmZTk5MzhiZmE2N2EzJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.AgFNSTnz4cBfbNVswN1TX5uZPWgn2-W3rrt9qjA4-bY) for the images has private-user-images in it the link, but it still works.
The Imgur link has pictures as well. You can right click those and "view image" to see the full size. https://imgur.com/a/ztgrLKG
I think you should have one part of the cable connected to the new board, two cables connected to the power supply, and the other to the backlight board. Then you have the LVDS cable. You should be able to message the seller and get a better image, but if you compare the pictures from our two ebay links, they look pretty similar. 
If you're unsure on this, maybe ask a local person for help doublechecking connections and going over whatever documents you can get before powering it on.

**Comment by tung256** [comment]
@jeremy-fields   happy to say i got the display working thanks to you! 
now onto getting the sound working. it seems i was overthinking or something. is connecting sound this simple???
![image](https://github.com/user-attachments/assets/97e33dd0-297d-4ab5-83cd-dc3ac0c573b2)

just take the 2 wires from each speaker and connect directly to the new board? that's the impression i get from @i3randnew  photos.

looks like we dont need this mic board at all
![image](https://github.com/user-attachments/assets/31916e5e-66b5-47a6-872d-08258d1816c5)

i believe this is the amplifier right?
![image](https://github.com/user-attachments/assets/8796c2e2-1424-4e35-a346-852a793c2d25)

but the new board has amplification built in? 

if my thinking is wrong, could you clarify with a schematics?

**Comment by jeremy-fields** [comment]
Yes, that is a mic board and it's not needed now. 
You're right about the audio cables. @i3randnew is talking about the connecters [here](https://github.com/olm3ca/mirror/issues/8#issuecomment-2094972987). I wired mine once to test it. I used the right wires but wrong connectors. It worked but the cables wouldn't stay on. @i3randnew's post refers to the right connectors. 
For the amp, that is it in the picture I believe. I wasn't sure how to wire it so I just plugged in the speakers. They worked. They would probably be louder with the amp. Not sure how loud or if you'd want the control board to adjust volume.

**Comment by tung256** [comment]
@jeremy-fields is the amp board 5,12, or 24v? i forgot. i unplugged the excess cable without noticing...
for now, i soldered the 4 speaker cables directly into the new board's L+ L- R+ R-. 
to use the amp with the new board, i believe this is the connection here
![image](https://github.com/user-attachments/assets/ac4ca7cc-41bf-4068-9589-5638d53c35b0)

white=left
black= ground
red= right.
will test this theory later

thanks for all your help!

**Comment by jeremy-fields** [comment]
I think I measured it at 12V https://imgur.com/dTp10rZ
Mine was tied to a 12V input on the old mainboard. I also think my wire colors were swapped too. Power for plus was on plus, and negative on negative, but red and black wire colors were swapped. Something to watch for.

**Comment by merjo1515** [comment]
@jeremy-fields I also have a Samsung but my Model is different. Will your instructions work for me? I'm not super tech savvy so I am hoping I can do this.
LTI400HN017QJG06A37G

**Comment by jeremy-fields** [comment]
@merjo1515 I'm not sure. There may be some mention of Samsung in the other issues attached to this repo.

**Comment by merjo1515** [comment]
> @merjo1515 I'm not sure. There may be some mention of Samsung in the other issues attached to this repo.

Wasn't yours a Samsung mirror?

**Comment by tung256** [comment]
@jeremy-fields  i just tried my theory as seen here and saw smoke came out, now my new controller board is dead. ouch.  
if anyone figured out how to use the amp, kindly let me know.

![image](https://github.com/user-attachments/assets/95ca8437-d9f7-47a4-8f3a-1bee95616706)

**Comment by jeremy-fields** [comment]
Oh no! 
I have some pictures of a red, white, and black cable coming from the mic board to that amp. I don't have an image of the back of the mic board, but it's not labeled. Not where my connection was at least. Mine was not connected where you are showing, that connection is labeled though and has a term PBTL that should be looked into.


![IMG_1780](https://github.com/user-attachments/assets/339f6a6d-590a-427b-8796-5ccec6b7b252)
![IMG_1763](https://github.com/user-attachments/assets/77e1e3af-81b3-4ebd-aa79-b7aa9297d252)
![IMG_1752](https://github.com/user-attachments/assets/6d3e8f13-ebe0-458d-adaf-20a5ab02b070)
![IMG_2029](https://github.com/user-attachments/assets/48d941fe-6cb0-4a8e-aa4e-371dcb1db496)

**Comment by tung256** [comment]
i dont want to burn another controller board. might as well buy a cheap amp with full control for volume, bass, and treble knobs.  nice that this takes 12v dc input too!
![image](https://github.com/user-attachments/assets/d28a0300-23ca-4e4c-bb70-af62aab387f0)

**Comment by minihadroncollider** [comment]
> @minihadroncollider great update! You've documented this well. For Airplay, I've been a fan of uxplay.
> 
> Can Google TV rotate orientation?

@olm3ca I'm trying out uxplay which works pretty well. I haven't worked out how to get it to load automatically at start-up though. Have you had any success with this?

As for Google TV, I believe you can side load an app which will rotate the orientation

**Comment by minihadroncollider** [comment]
@tung256 
Let me know how you get on with the amp. I found that although my speakers are working fne when connected directly, the volume is lower than I would like.

**Comment by tung256** [comment]
@minihadroncollider   i removed the red amplifier board. install another ampli that has bluetooth, usb, and aux.  the rpi connects to my Logi Media Server to access the music that way. all explained in this video  https://youtu.be/vJqoqVfDUQA

**Comment by pedropm32** [comment]
@olm3ca @avitascmor Thanks you so much! Instructions were so easy to follow. 

I ordered the board, replaced it, and now it works perfectly. I just left an HDMI out so I can connect anything I want. Currently, I'm using a Mac Mini, haha...

**Comment by tung256** [comment]
@minihadroncollider it's working great so far. i just installed a Solid State Relay since that video. that way, when the TV auto turn On, i dont hear that annoying click sound from the relay and another click when TV goes Off.   cant believe SSR exists. seems nobody cares about it to explain on youtube so i will do that on another video.
now, i still cant find an Android player that supports portrait mode. if you know, please let me know too

**Comment by ralewis85** [comment]
Any luck decoding the amplifier pins or finding a schematic?  I assume the input signal goes into the 3 wires next to resistors R16 and R17 on the bottom corner.

It looks like the pin. connector in the middle of the board is for Parallel Bridge Tied Load, though it was never populated in the factory setup so it doesn't make a lot of sense to populate it with an the after market board.

**Comment by sebastian-ch** [comment]
hello!  are we able to get the speakers working with just the new mainboard? Or do I need to get another ampli board

**Comment by blakedietz** [comment]
Can someone help verify that the item in the [original link](https://www.ebay.com/itm/166580220209?mkcid=16&mkevt=1&mkrid=711-127632-2357-0&ssspo=JMDajnaISMG&sssrc=2047675&ssuid=-kZm7FqJTmi&widget_ver=artemis&media=COPY) (which is broken) is the same as the item [here](https://www.ebay.com/itm/167573262901).

**Comment by olm3ca** [comment]
@blakedietz - thanks for sharing that. I'll update the link. It definitely is the same product but the vendor changed the name for some reason.

**Comment by ChandlerWelsh** [comment]
Hey, I have one of the controller boards that works with these older models. LM40SAMFHD700AG25WV.

I'm no longer doing this project, let me know if anyone wants it for cheap. Shipping from USA

**Comment by StoniestOceans** [comment]
@ChandlerWelsh if you still have it, I will take it.

**Comment by ChandlerWelsh** [comment]
I do shoot me an email at ***@***.***

**Comment by StoniestOceans** [comment]
@ChandlerWelsh  your email is **** out. Can't see it.

**Comment by ChandlerWelsh** [comment]
My username plus the google mail

**Comment by ragley** [comment]
Anyone have any issues getting display images to appear? I bought [this](https://www.ebay.com/itm/166624031452) board, and the backlight works properly, going on and off when input signal is detected by the board. Tested with HDMI and DVI. @jeremy-fields Was there something you did to get the picture to come in better with the button board? Pressing menu button doesn't make anything show up on the screen either though.

My cables are identical to https://github.com/olm3ca/mirror/issues/8#issuecomment-2316595362. Bad LVDS cable or defective board? I tried with the new cable and the old original one as well. 

Any help greatly appreciated.

**Comment by zarif98** [comment]
Made my own GitHub page with the work I did for making the mirror work: https://github.com/zarif98/lululemon-mirror-project

I think I have mostly everything working, including speakers and now I want to get a Mac Mini and take FT calls on it, but let's see.

**Comment by zabdo34** [comment]
Hey @zarif98 I am having a hard time installing your setup from github, it is exactly what I was looking to do on mine, also how did you get the ddc/ci to turn on and off the display?

**Comment by zarif98** [comment]
> Hey [@zarif98](https://github.com/zarif98) I am having a hard time installing your setup from github, it is exactly what I was looking to do on mine, also how did you get the ddc/ci to turn on and off the display?

Hi! I put it in the github description but it's basically just DDC/CI via ddcutil. This will also allow you to control the mirror brightness and everything using the remote as well.

**Comment by karlbfer** [comment]
EDIT: Instead of outputting audio from the tvboard, I just connected the aux cable directly to the miniPC. Buzz solved and speakers are phenomenal :)

Hey ya'll sick write up - I was able to get my mirror up and running with an old mini pc :) Was able to even hook up the speakers by using the new controller boards audio output via aux -> amplifier. There is a bit of a "bzzzz" going on through the speakers though, I wonder if its a grounding or interference issue :( Ground loop isolator w/ tv controller board did not solve the issue 

@zarif98 I saw that you were able to get your speakers up nice and running too. Do you experience the same "bzzzz" sound as well? (nvm solved sorry for the ping!)
